assessment.js

```diff
@@ -1,8 +1,8 @@
 //We'll learn about require later in the course
 var ask = require('./ask.js');
 
-var answer = 'not empty';
+var answer = 0;
 
-while (answer !== '' && answer !== 'SeCrEt') {
-  answer = ask("Guess my secret? ");
+while (answer > 10 || answer < 1) {
+  answer = ask("Guess within the range of my secret number. ");
```

Comment:
I think the question actually meant that you should have a secret number between 1 and 10, and that people should try to guess it. But this solution is otherwise reasonable.